AppSyncRealTimeClient/Support/SubscriptionConstants.swift

```diff
@@ -7,21 +7,23 @@
 
 import Foundation
 
-public struct SubscriptionConstants {
+public enum SubscriptionConstants {
```

Comment:
looks like this was done automatically through the swift formatter/linter? Should we keep this change? If we do, is there any way this is a breaking change to our consumers (APIPlugin and AppSyncSDK) ?

Reply:
This looks like a non-breaking change because all callers are only accessing the static strings from this class